vault/data_source_aws_access_credentials_test.go

```diff
@@ -82,26 +84,28 @@ data "vault_aws_access_credentials" "test" {
 }`, mountPath, accessKey, secretKey)
 }
 
-func testAccDataSourceAWSAccessCredentialsConfig_sts(mountPath, accessKey, secretKey string) string {
+func testAccDataSourceAWSAccessCredentialsConfig_sts(mountPath, accessKey, secretKey, ttl string) string {
 	return fmt.Sprintf(`
 resource "vault_aws_secret_backend" "aws" {
     path = "%s"
     description = "Obtain AWS credentials."
     access_key = "%s"
-    secret_key = "%s"
+		secret_key = "%s"
 }
 
 resource "vault_aws_secret_backend_role" "role" {
     backend = "${vault_aws_secret_backend.aws.path}"
     name = "test"
-    policy = "{\"Version\": \"2012-10-17\", \"Statement\": [{\"Effect\": \"Allow\", \"Action\": \"iam:*\", \"Resource\": \"*\"}]}"
+		policy = "{\"Version\": \"2012-10-17\", \"Statement\": [{\"Effect\": \"Allow\", \"Action\": \"iam:*\", \"Resource\": \"*\"}]}"
+
 }
 
 data "vault_aws_access_credentials" "test" {
     backend = "${vault_aws_secret_backend.aws.path}"
     role = "${vault_aws_secret_backend_role.role.name}"
-    type = "sts"
-}`, mountPath, accessKey, secretKey)
+		type = "sts"
+		ttl  = "%s"
```

Comment:
Would it be possible to have both a positive and a negative test for the TTL? One with it and one without it ensuring that it's simply not passed on to Vault when it's not explicitly set by the user?